production.yml

```diff
@@ -3,6 +3,9 @@ version: '3'
 volumes:
   production_postgres_data: {}
   production_postgres_data_backups: {}
+  static_files:
+    name: django-static
```

Comment:
This is the name that's reflected on the host, so it should be something like `pra-request-tracker-static`